src/page-multi-facility/MultiFacilityImpactDashboard.tsx

```diff
@@ -137,39 +139,46 @@ const MultiFacilityImpactDashboard: React.FC = () => {
   const showRateOfSpreadTab = useFlag(["showRateOfSpreadTab"]);
   return (
     <MultiFacilityImpactDashboardContainer>
-      {scenario.loading ? (
-        <Loading />
-      ) : (
-        <ScenarioSidebar numFacilities={facilities?.data.length} />
-      )}
-      <div className="flex flex-col flex-1 pb-6 pl-8 justify-start">
-        <div className="flex flex-row flex-none justify-between items-start">
-          <AddFacilityButton onClick={openAddFacilityPage}>
-            <IconAdd alt="add facility" src={iconAddSrc} />
-            <AddFacilityButtonText>Add Facility</AddFacilityButtonText>
-          </AddFacilityButton>
-          <ScenarioTabs>
-            <ScenarioTabList>
-              <ScenarioTab
-                active={selectedTab === 0}
-                onClick={() => setSelectedTab(0)}
-              >
-                <TextLabel padding={false}>Projections</TextLabel>
-              </ScenarioTab>
-              {showRateOfSpreadTab ? (
+      <ToastProvider
+        placement="bottom-center"
+        transitionDuration={0}
+        components={{ Toast }}
+      >
```

Comment:
since we are using the same configuration everywhere I think you could just wrap the entire app in a `ToastProvider` rather than having to do it page by page .... we do that in `SiteProvider`

Reply:
I had tried to add it in SIteProvider earlier, but I couldn't get it to work for some reason. It seems to be working now, so moved it to SiteProvider.